index.bs

```diff
@@ -230,6 +230,9 @@ of the experience if it is website initiated. However, user agent MAY expose
 Picture-in-Picture window controls that change video playback state (e.g.
 pause).
 
+As one of the <a>unloading document cleanup steps</a>, run the <a>exit
+Picture-in-Picture algorithm</a>.
```

Comment:
It would be significantly better to actually add that step to the algorithm in HTML, and only include a note here saying that this is called from that point

Reply:
It sounds that these steps were designed to have API hook into them, right? Is it really worth having part of the Picture-in-Picture API behoviour defined in HTML?

Reply:
For info, the fullscreen API hooks the same way into it: https://fullscreen.spec.whatwg.org/#model (look for `unloading document cleanup steps`). @yoavweiss WDYT?

Reply:
OK, fair enough